SCClassLibrary/Common/Collections/Event.sc

```diff
@@ -44,17 +47,9 @@ Event : Environment {
 		//		^this.delta
 	}
 
-	// this[\isRest] may be nil
 	isRest {
-		^this[\isRest] == true
-		or: { this[\type] == \rest
-			or: {
-				this.use {
-					parent ?? { parent = defaultParentEvent };
-					~detunedFreq.value.isRest
-				}
-			}
-		}
+		_Event_IsRest
+		^false
```

Comment:
In my benchmarks, the `^false` fallback added 0.009 seconds to each 100,000 iteration test... so yes, it's faster to put the return in the primitive, but it's less of a speed impact than I measured for the repeated symbol lookups.

No argument on it being cleaner. I'm ok to change it as you suggest.

I had asked about this specific point before committing, and no objections were raised at that time...